packages/mapviewer/src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -125,18 +128,16 @@ function getIframeHosts(value) {
                     :external-iframe-hosts="externalIframeHosts"
                     :title="key"
                 />
-                <div
-                    v-else
-                    class="fw-bold"
-                >
-                    {{ t(key) }}
+                <div class="d-flex flex-row justify-content-between">
+                    <div class="fw-bold">{{ t(key) }}</div>
+                    <!-- eslint-disable vue/no-v-html-->
+                    <div
+                        data-cy="feature-detail-description-content"
+                        class="text-end pr-3"
+                        v-html="t(value)"
```

Comment:
why switching to a `v-html` for feature details?
that might create new security issues with external layers (possible phishing vector)

Reply:
I did not switch to `v-html`, it was already a `v-html` and I added a flexbox around the two divs to have them on a horizontal layout and added a translation around it. This is already used as a v-html for our external layers.